pkg/graveler/model.go

```diff
@@ -222,15 +217,19 @@ func ProtoFromRepositoryMetadata(metadata RepositoryMetadata) *RepoMetadata {
 	}
 }
 
-func PullRequestFromProto(pb *PullRequestData) *PullRequest {
-	return &PullRequest{
-		CreationDate: pb.CreatedAt.AsTime(),
-		Status:       pb.Status,
-		Title:        pb.Title,
-		Author:       pb.Author,
-		Description:  pb.Description,
-		Source:       pb.SourceBranch,
-		Destination:  pb.DestinationBranch,
+func PullRequestFromProto(pb *PullRequestData) *PullRequestRecord {
```

Comment:
What's different? Why this returns `Record` while all other `fromProto` funcs return non-Record objects?

Reply:
See: `TagFromProto`